x/emissions/keeper/inference_synthesis/synth_palette_forecast_implied_test.go

```diff
@@ -110,7 +110,7 @@ func (s *InferenceSynthesisTestSuite) TestCalcForecastImpliedInferencesTwoWorker
 	}
 
 	expected := map[string]*emissionstypes.Inference{
-		"forecaster0": {Value: alloraMath.MustNewDecFromString("1.018004876148456175753781623")},
+		"forecaster0": {Value: alloraMath.MustNewDecFromString("1.055841253742177320400327600231111")},
```

Comment:
where are these random long numbers coming from? simulation csv?

Reply:
Just recalculated after we've made the change in the std dev